src/sentry/static/sentry/app/views/onboarding/index.jsx

```diff
@@ -29,8 +30,11 @@ const OnboardingWizard = React.createClass({
       next: this.next,
       platform: this.state.platform,
       setPlatform: p => {
-        if (!this.state.projectName || this.state.platform === this.state.projectName) {
-          this.setState({projectName: p});
+        if (
+          !this.state.projectName ||
+          parseLastToken(this.state.platform) === this.state.projectName
```

Comment:
Let's use name, like "Node.js", instead of shortening the platform id, "like nodejs".